kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/admission/AdmissionRequest.java

```diff
@@ -0,0 +1,520 @@
+/**
```

Comment:
I know this file was copy pasted and modified from code generation, but it really looks messy with empty comments and partial javadoc.
Is it possible to clean this up a little?
Also use diamond operator where appropriate, etc.